### ISSUE TITLE
text2image & 'Bash on Windows'

### ISSUE THREAD

**Issue by amitdo** [amitdo/text2tif#13]
@Shreeshrii,

Hi !
Does text2image works on 'Bash on Windows'?

**Comment by Shreeshrii** [comment]
Yes, I use it on windows 10. Initially I used directly in 'Bash on Windows'. Now I use it via mobaxterm.

Basically it provides lubuntu 14.04 environment.

**Comment by Shreeshrii** [comment]
I was checking just now to see whether the .exe files from artifacts would work under 'git bash' which is more lightweight and could run the tesstrain.sh script with the windows exe files.

**Comment by Shreeshrii** [comment]
https://git-for-windows.github.io/

https://msdn.microsoft.com/en-us/commandline/wsl/about

http://mobaxterm.mobatek.net/download-home-edition.html

**Comment by amitdo** [comment]
Thanks. Good to know.

**Comment by Shreeshrii** [comment]
Just to clarify, it is the 'ubuntu' version of text2image that works well under bash on windows.

The windows version of text2image eg. provided by Stefan works on windows 'cmd' but I have had trouble completing the training process with the generated files. Also, windows cannot directly run the bash scripts.

I read about git bash today and wanted to try using the tesstrain.sh bash scripts directly in it with the latest .exe files from appyeyor artifacts, Don't know yet whether the training bash scripts will work.

**Comment by amitdo** [comment]
Does the bash script works on '**Bash** on Windows'?

**Comment by Shreeshrii** [comment]
yes, bash script works on 'Bash on Windows'.

>Bash on Windows provides developers with a familiar Bash shell and Linux environment in which you can run most Linux command-line tools, directly on Windows, UNMODIFIED, without needing an entire Linux virtual machine!

**Comment by amitdo** [comment]
Yes, I know that bash scripts should work on WSL, but I wanted to know if you tested tesstrain.sh on WSL.

**Comment by Shreeshrii** [comment]
Yes, WSL is what I have been using for last few months for all the tests for training. So, yes tesstrain.sh works on it.
